app/src/main/java/eu/kanade/presentation/manga/components/MangaAppBar.kt

```diff
@@ -55,6 +55,8 @@ fun MangaAppBar(
     onEditInfoClicked: () -> Unit,
     showRecommends: Boolean,
     onRecommendClicked: () -> Unit,
+    showMerge: Boolean,
+    onMergeClicked: () -> Unit,
```

Comment:
Instead of a boolean make onMergeClicked nullable `onMergeClicked: (() -> Unit)?`

Reply:
https://github.com/jobobby04/TachiyomiSY/pull/653/commits/69e7a1a229dcbe58e439f5fc88262835a930b338